content/en/docs/zero-code/java/spring-boot-starter/getting-started.md

```diff
@@ -26,8 +26,8 @@ A Bill of Material
 ensures that versions of dependencies (including transitive ones) are aligned.
 
 To ensure version alignment across all OpenTelemetry dependencies, you must
-import the `opentelemetry-instrumentation-bom` BOM when using the OpenTelemetry
-starter.
+import the `opentelemetry-instrumentation-bom-alpha` BOM when using the
+OpenTelemetry starter.
```

Comment:
this seems like an unfortunate workaround for a conflict between two libraries that shouldn't be used together anyways (otel spring boot starter and micrometer tracing bridge)

if we're going to go with this change, I think we should add some kind of explanation why the `-alpha` bom is needed for the stable otel spring boot starter